### PR TITLE
Merge master into release

### PR DESCRIPTION
**Pull request by melvinsoft**
I created a PR for merge `appsembler/master` into `appsembler/release` as a first step, then we need to merge `appsembler/release` into `appsembler/eucalyptus/develop` and `appsembler/dogwood/develop`. I created PR because we have some changes for testdrive here, and I'm not familiar at all with the project.

### CODE REVIEW

File: playbooks/amc.yml [hunk 1]
```diff
@@ -30,9 +30,9 @@
     - role: mongo
       when: "'localhost' in EDXAPP_MONGO_HOSTS"
     - { role: 'rabbitmq', rabbitmq_ip: '127.0.0.1' }
+    # Do we need need edxapp 2 times here????
```
Comment (johnbaldwin) [inline playbooks/amc.yml]: @mrgnr @denibertovic Can you answer this one? I don't know

File: playbooks/roles/backups/defaults/main.yml [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
 
 # gs: Google Cloud Storage
 # s3: Amazon Simple Storage Service
 BACKUPS_PROVIDER: 'gs'
 
+# AWS
+BACKUPS_AWS_ACCESS_KEY_ID: ''
+BACKUPS_AWS_SECRET_ACCESS_KEY: ''
 
 # Google Cloud Storage
```
Comment (johnbaldwin) [inline playbooks/roles/backups/defaults/main.yml]: @melvinsoft Maybe group the Google Cloud storage together? (see above)

**Comment by johnbaldwin** [comment]
@melvinsoft Done reviewing. I commented on the two comments you made on multiple calls to the same role, I pinged/tagged Deni, Filip, Morgan on those

File: playbooks/roles/backups/defaults/main.yml (hunk 2 above)
Comment (melvinsoft) [inline playbooks/roles/backups/defaults/main.yml]: @johnbaldwin those two are for AWS, not for Google Cloud Storage. I think are well grouped already.

**Pull request merged**